recipes/folly/all/patches/0017-std-hash.patch

```diff
@@ -0,0 +1,180 @@
+diff --git a/folly/Uri-inl.h b/folly/Uri-inl.h
```

Comment:
This change is also submitted to folly itself: https://github.com/facebook/folly/pull/1587